.hlint.yaml

```diff
@@ -0,0 +1,10 @@
+# HLint configuration file
+# https://github.com/ndmitchell/hlint
+##########################
+
+- ignore: {name: "Redundant multi-way if"}
+- ignore: {name: "Redundant do"}
+- ignore: {name: "Reduce duplication"}
+- ignore: {name: "Eta reduce"}
```

Comment:
I like to be reminded of these -- even though I often ignore those. Really, I think, hlint is a tool for personal use. It isn't a good replacement for a developers of projected converging towards a common coding style.